Based on mumble-developers-qt/4.8-mumble
00040b37cebb7192349fcb6c75667a828ab05cee

--- ./qtbase/src/network/ssl/qsslsocket_openssl.cpp
+++ ./qtbase/src/network/ssl/qsslsocket_openssl.cpp
@@ -1164,6 +1164,9 @@ bool QSslSocketBackendPrivate::startHand
         return false;
     }
 
+    // Drain error queue
+    while(q_ERR_get_error());
+
     // store peer certificate chain
     storePeerCertificates();
 
